ts/theme/components/View.ts

```diff
@@ -15,10 +15,19 @@ declare module "native-base" {
       header?: boolean;
       content?: boolean;
       padded?: boolean;
+      noPadded?: boolean;
     }
   }
 }
 
+/**
+ * TO DO: 
+ * if no components are inserted as footer, the following component should be 
```

Comment:
should this be done on each screen? can you add a story for it with a short explanation on what needs to be done? thx

Reply:
Can't proceed with lint error fix. The tslint command is not recognized. I have problems with yarn installation yet :(